docs/user-guide/area_calc.ipynb

```diff
@@ -6,9 +6,6 @@
    "source": [
```

Comment:
<p>These comments can be omitted. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
fixed. why doesn't it show [here](https://uxarray--788.org.readthedocs.build/en/788/userguide.html)